<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html> <head>
<title>Robert G. Rogers Jr: Resume</title>
<link href="../site.css" title="Default" rel="stylesheet" type="text/css">
</head>

<body bgcolor=white>
<h1>Robert G. Rogers Jr</h1>

<p><a href="/"><tt>Home</tt></a> : <a href="/bob/">Stuff about me</a> :
Resume (summary)
<hr>

<p><i>[This is the abbreviated version; I got tired of getting spammed
by head-hunters who couldn't be bothered even to skim it well enough to
know that I wasn't at all what they were looking for.  Besides, the full
version had fallen out of date.  -- rgr, 25-Dec-04.]</i>

<a name = expertise>
<h2>Summary of Expertise</h2>

<dl>
  <dt> <b>Software Engineering</b>
  <dd>
       <dl>
	 <dt> Interests
	 <dd>
	      <ul>
		<li> Applying object-oriented technology to rationalize
		     the design of large software systems.
		<li> System and network security.
	      </ul>

	 <dt> Systems
	 <dd> Linux, Solaris, SunOS, Tru64 Unix.
	 <dd> Five years sysadmin experience with Linux, and a
	      smattering with the others.

	 <dt> Languages (fluent)
	 <dd> Fluent:  Common Lisp, Perl, emacs lisp.
	 <dd> Proficient:  C, SQL, Unix shell.
	 <dd> Historic:  Basic, Fortran, PL/1, COBOL, various
	      assemblers.
       </dl>
  <dt> <b>Bioinformatics</b>
  <dd>
       <dl>
	 <dt> Interests
	 <dd>
	      <ul>
		<li> Protein structure modeling.
		<li> I have long been fascinated by evolution,
		     particularly early molecular evolution and the
		     emergence of life.
	      </ul>

	 <dt> Experience
	 <dd> Two years experience in writing software to automate oligo
	      design for PCR of various sorts.
	 <dd> Experienced with manipulation of the <a
	      href="http://www.rcsb.org/">Protein Data Bank</a> format
	      for publishing protein structures (see the <a
	      href="http://bmerc-www.bu.edu/needle-doc/new/atom-format.html">
	      <tt>http://bmerc-www.bu.edu/needle-doc/new/atom-format.html</tt>
	      </a> page, for example).
	 <dd> Experienced with using and extending sequence data
	      manipulation tools (e.g. search and alignment).
       </dl> 
</dl>

<a name = pubs>
<h2>Publications</h2>

<dl compact="compact">
  <dt>[1]</dt>
  <dd> M. Cline, K. Karplus, R.&nbsp;H. Lathrop, T.&nbsp;F. Smith,
       R.&nbsp;G. Rogers&nbsp;Jr., D. Haussler.  Information-theoretic
       dissection of pairwise contact potentials.  <em>Proteins:
       Structure, Function and Genetics</em>.  [Revised draft
       submitted.]
  <dt>[2]</dt>
  <dd> J. Bienkowska, H. He, R.&nbsp;G. Rogers Jr., and L. Yu.  Bayesian
       approach to protein fold recognition: building protein structural
       models from bits and pieces.  Chapter in Igor Tsigelny, editor,
       <em>Protein Structure Prediction: Bioinformatics approach</em>,
       2002, pages 43-84.  [In press.]
  <dt>[3]</dt>
  <dd> J.&nbsp;R. Bienkowska, L.&nbsp;H. Yu, R.&nbsp;G. Rogers Jr.,
       S. Zarakhovich, and T.&nbsp;F. Smith.  Protein fold recognition
       by total alignment probability.  <em>Proteins: Structure,
       Function and Genetics</em>, 40:451-462, 2000.
  <dt>[4]</dt>
  <dd> J.&nbsp;R. Bienkowska, R.&nbsp;G. Rogers Jr., and
       T.&nbsp;F. Smith.  Eliminating superfluous neighbor pairs while
       threading fold models.  In <em>Pacific Symposium on Biocomputing
       2000</em>, January 2000.
  <dt>[5]</dt>
  <dd> J.&nbsp;R. Bienkowska, R.&nbsp;G. Rogers Jr., and
       T.&nbsp;F. Smith.  A method for optimal design of a threading
       scoring function.  In S.&nbsp;Istrail, P.&nbsp;Pevzner, and
       M.&nbsp;Waterman, editors, <em>RECOMB99 Proceedings of the Third
       Annual International Conference on Computational Molecular
       Biology</em>, pages 25-32, Lyon, France, April 1999. ACM Press
       New York NY.
  <dt>[6]</dt>
  <dd> J.&nbsp;R. Bienkowska, R.&nbsp;G. Rogers Jr., and
       T.&nbsp;F. Smith.  Performance of threading scoring function
       designed using a new optimization method.  <em>Journal of
       Computational Biology</em>, 6(3):299-311, 1999.
  <dt>[7]</dt>
  <dd> J.&nbsp;R. Bienkowska, R.&nbsp;G. Rogers Jr., and
       T.&nbsp;F. Smith.  Filtered neighbors threading.  <em>Proteins:
       Structure, Function and Genetics</em>, 1999.
  <dt>[8]</dt>
  <dd> R.&nbsp;H. Lathrop, R.&nbsp;G. Rogers Jr., J.&nbsp;R. Bienkowska,
       B.&nbsp;K.&nbsp;M. Bryant, L.&nbsp;J. Buturovic,
       C.&nbsp;Gaitatzes, R.&nbsp;Nambudripad, J.&nbsp;V. White, and
       T.&nbsp;F. Smith.  "Analysis and Algorithms for Protein
       Sequence-Structure Alignment," chapter in <em>Computational
       Methods in Molecular Biology</em>, Elsevier Press, Amsterdam,
       Netherlands, 227-283, 1998.
  <dt>[9]</dt>
  <dd> R.&nbsp;H. Lathrop, R.&nbsp;G. Rogers Jr., T.&nbsp;F. Smith, and
       J.&nbsp;V. White.  A bayes-optimal sequence-structure theory that
       unifies protein sequence-structure recognition and alignment.
       <em>Bull. Math. Biol.</em>, 60:1039-1071, 1998.
  <dt>[10]</dt>
  <dd> T.&nbsp;F. Smith, L.&nbsp;Lo Conte, J.&nbsp;Bienkowska,
       C.&nbsp;Gaitatzes, R.&nbsp;G. Rogers Jr., and R.&nbsp;H. Lathrop.
       Current limitations to protein threading approaches.
       <em>J. Comp. Biol.</em>, 4:217-225, 1997.
  <dt>[11]</dt>
  <dd> S.&nbsp;Das, L.&nbsp;H. Yu, C.&nbsp;Gaitatzes, R.&nbsp;Rogers,
       J.&nbsp;Freeman, J.&nbsp;Bienkowska, R.&nbsp;M. Adams,
       T.&nbsp;F. Smith, and J.&nbsp;Lindellen.  Biology's new rosetta
       stone.  <em>Nature</em>, 385:29-30, 1997.
  <dt>[12]</dt>
  <dd> T.&nbsp;F. Smith, L.&nbsp;Lo Conte, J.&nbsp;R. Bienkowska,
       R.&nbsp;G. Rogers Jr., C.&nbsp;Gaitatzes, and R.&nbsp;H. Lathrop.
       The threading approach to the inverse protein folding problem.
       In S.&nbsp;Istrail, P.&nbsp;Pevzner, and M.&nbsp;Waterman,
       editors, <em>RECOMB97 Proceedings of the First Annual
       International Conference on Computational Molecular Biology</em>,
       New York, NY, January 1997. ACM Press.
</dl>

<p>
<hr>
<address>Bob Rogers <tt>&lt;rogers@rgrjr.dyndns.org&gt;</tt></address>
$Id$
</body>
</html>
